apps/src/lib/tools/jsdebugger/JsDebugger.jsx

```diff
@@ -406,7 +425,7 @@ export const UnconnectedJsDebugger = Radium(React.createClass({
         >
           <span
             style={[
-              this.state.windowWidth <= 1275 && !this.state.watchersHidden && styles.hidden,
+              this.state.consoleWidth <= 345 && styles.hidden,
```

Comment:
We start it being hidden by default? (based on the fact that we set console width to 345, and use <=).